spec/puppet-server-lib/puppet/jvm/puppet_config_spec.rb

```diff
@@ -17,7 +17,7 @@
     describe "the puppet log level (Puppet::Util::Log.level)" do
       subject { Puppet::Util::Log.level }
       it 'is set to debug (the highest) so messages make it to logback' do
-        expect(subject).to eq('debug')
+        expect(subject).to eq(:debug)
```

Comment:
Appears to be due to the same `Symbol` -> `String` comparison issue mentioned earlier.